### DISCUSSION TITLE
why did my github website stop showing up

### DISCUSSION THREAD

**Discussion by Idossou-Marius-Adom** [community/community#143215] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello. I am not a proficient Github programmer, but I managed to create a github website more than a year ago. It was functionning well and I used to update it from time to time. But since two weeks now it stopped showing up. No error message and the link to the home page functions. But on the home page it is blank: nothing shows up except a picture. This didn't happen after any particular action, it just happened. I searched the files and everything seems correct (I mean as before when it was well functionning). Does someone understand what is going on and how to solve the issue? Thank you.

**Comment by LiteBrite82** [comment]
Hi @Idossou-Marius-Adom 
Sorry to hear your page isn't working correctly. Can you share a link to the repo and deployment URL for the community to better assist?

**Comment by Idossou-Marius-Adom** [comment]
Hi @LiteBrite82. Thank you very much for helping. The repo is here https://github.com/Idossou-Marius-Adom and the deployment here https://idossou-marius-adom.github.io/.
